lib/analyzer/index.ts

```diff
@@ -67,6 +83,7 @@ async function analyze(
     osRelease,
     results,
     binaries,
+    hashes: await hashAnalyzer.analyze(docker),
```

Comment:
let's do the same as we're doing for `binaries` - initialize separately and wrap with try & catch